.github/infrastructure/docker-compose-postgresql.yml

```diff
@@ -0,0 +1,11 @@
+version: '2'
+services:
+  db:
+    image: postgres
+    restart: always
+    ports:
+      - "5432:5432"
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: example
+      POSTGRES_DB: dapr_test
```

Comment:
Nit: add newline at end of file for Linux text files.

Reply:
done.